YAML/YAML.sublime-syntax

```diff
@@ -527,16 +527,23 @@ contexts:
             - match: '{{_flow_scalar_end_plain_out}}'
               pop: true
     - match: :(?=\s|$)
-      scope: punctuation.separator.mapping.key-value.yaml
+      scope: punctuation.separator.key-value.mapping.yaml
 
   comment:
     # http://www.yaml.org/spec/1.2/spec.html#comment//
     - match: | # l-comment
         (?x)
-        (?: ^ [ \t]* | [ \t]+ )
-        (?:(\#) \p{Print}* )?
-        (\n|\z)
-      scope: comment.line.number-sign.yaml
+        (?: ( ^ [ \t]* ) | [ \t]+ )
+        (?=#\p{Print}*$)
```

Comment:
Additionally, this regex pretty much hoses the YAML definition.

![screen shot 2016-09-21 at 5 04 05 am](https://cloud.githubusercontent.com/assets/41734/18704692/06d5fd32-7fb9-11e6-9b44-faf7a8fb1738.png)